SwedbankPaySDKTests/FileLinesTests.swift

```diff
@@ -0,0 +1,78 @@
+import XCTest
+import Foundation
+@testable import SwedbankPaySDK
+
+class FileLinesTests: XCTestCase {
+    private func checkThatFile(with content: String, parsesTo lines: String...) throws {
+        let file = try FileUtils.openString(content)
+        defer {
+            fclose(file)
+        }
+        file.getLines().assertElementsEqual(lines)
+    }
+    
```

Comment:
Codacy found an issue: [Lines should not have trailing whitespace.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-ios/pullRequest?prid=6137886)

Reply:
Codacy is being weird here. These lines are identical to other single blank lines in the same files. Ignoring these for now.